<launch>
    <arg name="config_file" default = "/home/javens/Workspace/catkin_dio/src/ROGIO-ROS-threads/config/Kinect_lab_config.yaml" />


    <node name="rogio_threads" pkg="rogio_threads" type="rogio_threads" output="screen">
        <param name="config_file" type="string" value="$(arg config_file)" />

    </node>
</launch>
